public-api/src/main/resources/public-api.yaml

```diff
@@ -324,7 +325,14 @@ definitions:
           type: "array"
           items:
             $ref: "#/definitions/Concept"
-
+        matchType:
+          type: string
```

Comment:
Consider making this an enum instead of a String.

Reply:
Done